src/NServiceBus.AcceptanceTesting/FodyWeavers.xsd

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should instead remove Fody.Janitor and Fody and remove the [SkipWeaving] directive from AcceptanceTestingOutboxTransaction